stustapay/core/schema/example_data/example_data.sql

```diff
@@ -36,53 +36,53 @@ values
 
 
 insert into product (
-    id, name, price, target_account, tax
+    id, name, price, tax
 )
 values
     -- Getränke
-    (0, 'Helles 1.0l', 4.2016806722, null, 'ust'), -- brutto price 5.00
-    (1, 'Helles 0.5l', 2.5210084033, null, 'ust'), -- brutto price 3.00
-    (2, 'Weißbier 1.0l', 4.2016806722, null, 'ust'),
-    (3, 'Weißbier 0.5l', 2.5210084033, null, 'ust'),
-    (4, 'Radler 1.0l', 4.2016806722, null, 'ust'),
-    (5, 'Radler 0.5l', 2.5210084033, null, 'ust'),
-    (6, 'Russ 1.0l', 4.2016806722, null, 'ust'),
-    (7, 'Russ 0.5l', 2.5210084033, null, 'ust'),
-    (8, 'Limonade 1.0l', 1.6806722689, null, 'ust'), -- brutto price 2.00
+    (0, 'Helles 1.0l', 4.2016806722, 'ust'), -- brutto price 5.00
+    (1, 'Helles 0.5l', 2.5210084033, 'ust'), -- brutto price 3.00
+    (2, 'Weißbier 1.0l', 4.2016806722, 'ust'),
+    (3, 'Weißbier 0.5l', 2.5210084033, 'ust'),
+    (4, 'Radler 1.0l', 4.2016806722, 'ust'),
+    (5, 'Radler 0.5l', 2.5210084033, 'ust'),
+    (6, 'Russ 1.0l', 4.2016806722, 'ust'),
+    (7, 'Russ 0.5l', 2.5210084033, 'ust'),
+    (8, 'Limonade 1.0l', 1.6806722689, 'ust'), -- brutto price 2.00
```

Comment:
hm maybe we should store the price including tax.. so we can easier get the desired real output price.